coders/heic.c

```diff
@@ -822,128 +985,97 @@ static MagickBooleanType WriteHEICImage(const ImageInfo *image_info,
   heif_context=heif_context_alloc();
   heif_image=(struct heif_image*) NULL;
   heif_encoder=(struct heif_encoder*) NULL;
+
   do
   {
-    const Quantum
-      *p;
-
 #if LIBHEIF_NUMERIC_VERSION >= 0x01040000
     const StringInfo
       *profile;
 #endif
 
-    int
-      stride_y,
-      stride_cb,
-      stride_cr;
-
     struct heif_error
       error;
 
     struct heif_writer
       writer;
 
-    uint8_t
-      *p_y,
-      *p_cb,
-      *p_cr;
+    enum heif_colorspace 
+      colorspace;
+
+    enum heif_chroma 
+      chroma;
+
+    MagickBooleanType 
+      isAvif;
```

Comment:
We use underscores instead of camel case. `isAvif => is_avif`.

Reply:
Yer, sorry, switching between Java and Go, then C at night... fixed it